### PR TITLE
Create `cogAtlas` namespace and update examples + example README

### LINKED ISSUE #106: Create namespace for Cognitive Atlas terms
- [x] Update `field_preamble` for the context in pheno_utils.py 
https://github.com/neurobagel/bagel-cli/blob/d5762fc508a469b6f87f1745eb7fd2e9bb3263d4/bagel/pheno_utils.py#L15-L22
  with `"cogAtlas": "https://www.cognitiveatlas.org/task/id/"`
- [x] update context of `example_synthetic.jsonld`

### LINKED ISSUE #116: Document `synthetic` example in test data README

### PR DESCRIPTION
**Pull request by alyssadai**
Closes #106
Closes #116

### CODE REVIEW

File: bagel/pheno_utils.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def generate_context():
         "bg": "http://neurobagel.org/vocab/",
         "snomed": "https://identifiers.org/snomedct:",
         "nidm": "http://purl.org/nidash/nidm#",
+        "cogAtlas": "https://www.cognitiveatlas.org/task/id/",
```
Comment (surchs) [inline bagel/pheno_utils.py:22]: now that we're trying to be consistent with this, we should have a central place for namespace conventions (and also for names for properties/edges and classes/nodes). I'm thinking such a place doesn't exist yet (does it?) - would you mind creating an issue to create such a place?

File: bagel/tests/data/README.md [hunk 2]
```diff
@@ -15,5 +15,6 @@ Example inputs to the CLI
 | 8       | valid, based on ex2 has multiple participant_id columns            | valid, based on ex2 multiple participant_id column annotations                   | fail*  |
 | 9       | invalid, based on example 6 but contains an unannotated value for `group`            | valid, based on example 6                   | fail  |
 | 10       | valid, same as example 6           | valid, based on example 6 but contains extra `"MissingValues"` not found in the .tsv                   | pass, with warning  |
+| synthetic       | valid, has `participant` and `session` IDs corresponding to the [`synthetic` example BIDS dataset](https://github.com/bids-standard/bids-examples/tree/master/synthetic)           | valid                   | pass  |
```
Comment (surchs) [inline bagel/tests/data/README.md:18]: all good, but the markdown is looking sad. I'm using a [pycharm functionality for aligning this](https://www.jetbrains.com/help/pycharm/markdown.html#table), maybe there is a similar plugin for vscode?

Comment (surchs) [reply bagel/tests/data/README.md:18]: not for this PR though

**Review by surchs — APPROVED** [review]
Looks good! Thanks @alyssadai. :cook: 

I left 2 train-of-thought comments (:steam_locomotive: ?), but nothing to be done here for this PR

File: bagel/tests/data/README.md (hunk 2 above)
Comment (alyssadai) [reply bagel/tests/data/README.md:18]: Ah yes, the Prettier extension can do this - I'll make sure to run it next time. I've realized also that when line wrapping is enabled even formatted tables look super wonky.

File: bagel/pheno_utils.py (hunk 1 above)
Comment (alyssadai) [reply bagel/pheno_utils.py:22]: @surchs I think the relevant issue is #65, specifically the first subtask. Worth separating that into its own issue?

**Pull request merged**